app/boards/shields/babyvble/babyvble.zmk.yml

```diff
@@ -0,0 +1,8 @@
+file_format: "1"
+id: babyvble
+name: BabyvBle
+type: shield
+url: https://discord.gg/ktUDJ3w
```

Comment:
I'm not sure how I feel about having a Discord invite link used as the information link for a keyboard. Do you have an update on how you're distributing keyboards like this BabyvBle? Ideally some type of group buy or online storefront presence?

Reply:
At the moment I don't have a storefront, my discord server is the main place I post updates about the board so for me it made the most sense to have that as the url. I am totally fine with removing the discord link and I can work on a different way to store information about my keyboards. I could set up a repo where I post information and updates about the board.

Reply:
I have made a [Github repo](https://github.com/MakerJake01/MakerJakes-keyboards/tree/main/BabyvBle) where I am storing information about the keyboard. I will replace the link in the PR with it.